### DISCUSSION TITLE
gouravkhunger/pagination-in-android-room-database-using-the-paging-3-library

### DISCUSSION THREAD

**Discussion by giscus** [genicsblog/comments#126] [Announcements]
# gouravkhunger/pagination-in-android-room-database-using-the-paging-3-library

The complete guide to paginating *just* a room database (without Network level pagination) with the Paging 3 library in Android using Kotlin.

https://genicsblog.com/gouravkhunger/pagination-in-android-room-database-using-the-paging-3-library

**Comment by Tabadia** [comment]
Very cool!

**Reply by gouravkhunger** [reply]
Thank you! 😄

**Comment by AppsDevMaruf** [comment]
Just Awesome <3

**Reply by gouravkhunger** [reply]
Thank you!
